pulser-core/pulser/noise_model.py

```diff
@@ -466,3 +508,50 @@ def from_abstract_repr(obj_str: str) -> NoiseModel:
                 obj_str
             )
         )
+
+    @staticmethod
+    def generate_detuning(
+        detuning_sigma: float,
+        detuning_high_freq: tuple[ArrayLike, ArrayLike],
+        times: ArrayLike,
+        rng: Generator = np.random.default_rng(),
+    ) -> ArrayLike:
+        """Compute δ_hf(t) + δ_σ.
```

Comment:
This method looks out of place here... I believe it will be moved in #916 anyway, so maybe just make it a function instead.

Reply:
That's true, but it's only temporary and will be handled in #916